adding_species_manuscript.tex

```diff
@@ -634,114 +641,112 @@ \subsection*{Extracting parameters from the literature}
-We caution that this has the potential to result in false precision when
-these effects are present in the real genome but missing from the diversity
-generated by the simulation.
-Finally, although whole-chromosome simulations are crucial for many purposes,
-for some situations it may be sufficient to rely on simulation of many unlinked sites \citep{Gutenkunst2009,Excoffier2013},
+%Ilan: I didn't quite get the point about "false precision", so I simplified this statement
+Creating pseudo-chromosomes allows the simulation to fit the amount of data of real genomes, but it artificially increases the correlation between variants.
+%
+Finally, we note that for some situations it may be sufficient to rely on simulation of a large number of unlinked sites \citep{Gutenkunst2009,Excoffier2013},
 which can be generated without any sort of genome assembly.
-However, we caution that in general the influence of linkage on the uncertainty
-of such inferences is not well understood.
-An alternative is to instead simulate an anonymous chromosome from which patterns
-of genetic variation can be extracted (if important, in chunks of size similar to the contigs).
-The latter is usually more realistic, since this includes linkage between sites that share a chromosome but
-may be on different real contigs.
-Precise locations in the simulated genomes cannot then be matched to particular contigs,
-but general statistical patterns can be compared.
-
-
-\begin{table}[htb]
-	\captionof{table}{Guide to missing parameters. \\
-	%
-	\colorbox{yellow}{[TODO: reconsider table. Either clarify considerations or just defer to the text, which covers this quite clearly.}
-	\colorbox{yellow}{Current version is too vague.}
-	%RNG: I vote to remove the table, which just repeats the text
-	%MEL: I vote table, easier to reference for someone truly using this as a guide.
-} \label{tab:param-mod}
-	\begin{tabular}{p{1.5in}p{2.2in}p{2.2in}}
-			\hline
-			Missing parameter  & Options & Considerations \\
-			\hline
-			Mutation rate      &
-			borrow from closest relative with a citeable mutation rate &
-			will affect levels of polymorphism \\
-            \hline
-			Recombination rate &
-			borrow from closest relative with a citeable rate &
-			will affect the impact of selection, linkage, and linked selection
-			\\
-			\hline
-			Demographic model &
-			at least Ne is required and is estimable from mutation rate and genetic data     &
-			the demographic history (e.g. bottlenecks, expansions, and population splits and migration) affects patterns of variation substantially {[}CITE{]}, a constant Ne is not ideal \\
-			\hline
-	\end{tabular}
-\end{table}
+% Ilan: shortened this last part and removed mention of the option to create anonymous chromosomes. IMO, for many purposes, it's probably better to keep the sites independent rather than create aritificial and erroneous linkage between them.
```

Comment:
I don't agree. By having a realistic amount of linkage you, for instance, would end up with a realistic amount of background selection acting across the genome (and so a simulation of unlinked loci would end up with too much genetic diversity). The precise landscape of background selection intensity would not be right, but the general statistical properties of it should be right-ish, to the extent that a random stitching together of contigs looks like a real chromosome.

Reply:
Okay. I'm making some more edits to this paragraph and will try to make this point in the end. My idea is to now describe cases with partial genome builds like this:
1) Treat long contigs as separate chromosomes [ main caveat is that you're missing out on linkage between contigs that map to same chrom ]
2) Lump together shorter contigs or omit them completely [ benefits and caveats to lumping ]
3) If the species 'genome build' only consists of short contigs, you may simulate artificial anonymous chroms. This (again) models linkage within each contigs and (approximately) captures some basic features such as BGS
4) Finally, you have the option of simulating independent sites, that you can do even without any genome build. however, then you miss out on the cool benefits of whole-chrom simulations, which is what we're trying to promote here

Reply:
from slack: maybe we need an example or two in this section; like "if you want to fit the distribution of LD values then it's better to simulate a whole anonymous genome; while if you want to compare the LD values for a specific piece of genome to simulation then it might be better to simulate that specific contig, either unlinked or linked to other random contigs (and take several draws of which random contigs it's linked to)"